run_mlp.py

```diff
@@ -0,0 +1,57 @@
+from argparse import ArgumentParser
+
+import numpy as np
+from pytorch_lightning import Trainer
+from torch.utils.data import DataLoader
+
+from dataloader import ChestXrayDataset, ChestXrayTestDataset
+from mlp import LitLinear
+
+BATCH_SIZE = 64
+PATCH_SIZE = 64
+N_PATCHES = 5
+
+
+def run_MLP():
+    parser = ArgumentParser()
+    parser = Trainer.add_argparse_args(parser)
+    args = parser.parse_args(["--gpus=0", "--max_epochs=1", "--max_steps=5", "--val_check_interval=40"])
```

Comment:
I do this too all the time for testing, so :man_shrugging:, but arguably, committed code should not do this. Instead, there should be e.g. a `test_mlp.sh` file that looks something like

```bash
source .venv/bin/activate
python run_mlp.py --gpus=0 --max_epochs=1 --max_steps=5 --val_check_interval=40
```
or the hardcoded args should go in a `tests/test_mlp.py` to be run with e.g. `pytest` or `nose` or `unittest` or whatever.